test/parallel/test-performance-uvmetricsinfo.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+require('../common');
+
+const fixtures = require('../common/fixtures');
+
+const filepath = fixtures.path('x.txt');
+
+const assert = require('node:assert');
+const fs = require('node:fs');
+const { performance } = require('node:perf_hooks');
+
+const { uvMetricsInfo } = performance;
+
+function safeMetricsInfo(cb) {
+  setImmediate(() => {
+    const info = uvMetricsInfo();
+    cb(info);
+  });
+}
+
+{
+  const info = uvMetricsInfo();
+  assert.strictEqual(info.loopCount, 0);
+  assert.strictEqual(info.events, 0);
+  // This is the only part of the test that we test events waiting
+  // Adding checks for this property will make the test flaky
+  // as it can be highly influenced by race conditions.
+  assert.strictEqual(info.eventsWaiting, 0);
+}
+
+{
+  // The synchronous call should obviously not affect the uv metrics
+  const fd = fs.openSync(filepath, 'r');
+  fs.readFileSync(fd);
+  const info = uvMetricsInfo();
+  assert.strictEqual(info.loopCount, 0);
+  assert.strictEqual(info.events, 0);
+  assert.strictEqual(info.eventsWaiting, 0);
+}
+
+{
+  function openFile(info) {
+    assert.strictEqual(info.loopCount, 1);
+    // 1. ? event
+    assert.strictEqual(info.events, 1);
+
+    fs.open(filepath, 'r', (err) => {
+      assert.ifError(err);
+      safeMetricsInfo(afterOpenFile);
+    });
+  }
+
+  function afterOpenFile(info) {
+    assert.strictEqual(info.loopCount, 2);
+    // 1. ? event
+    // 2. uv_fs_open
+    assert.strictEqual(info.events, 2);
+
+    fs.readFile(filepath, (err) => {
+      assert.ifError(err);
+      safeMetricsInfo(afterReadFile);
+    });
+  }
+
+  function afterReadFile(info) {
+    assert.strictEqual(info.loopCount, 6);
+    // 1. ? event
+    assert.strictEqual(info.events, 6);
+    // 1. ?
+  }
```

Comment:
Should we add at least a note about why the value is 6? Unless you think this will never change.

Reply:
Actually, due to how our test runs (requiring `common.js`) is a bit difficult to track exactly which events are these 6. My initial thought was to document exactly what are these 6 events, but, I decided to not dig into this rabbit hole, at least for now. Considering we're just relying on libuv for this operation, it shouldn't be a problem since any bug is possibly on the vendor.

This value might change depending on changes inside `common.js`, for instance, if it schedules a new I/O operation before this test, it will increase.

Reply:
Okay, at least with this comment people in the future will be able to have some guidance on what to do if this value changes.

Reply:
After struggling a bit with identifying events on AIX, I decided to just rely on libuv tests. If we increase our test coverage, specifically on this function, we'll eventually make this test flaky as there are several factors that would change the value of `events` and `eventsWaiting` (such as thread-schedule). 391ce64ee4355f07f5efaea07eb9ccb5ce40d99f